tests/acceptance/features/apiSpaces/shareOperations.feature

```diff
@@ -224,4 +224,168 @@ Feature: sharing
+    Then the HTTP status code should be "201"
+    And the following headers should match these regular expressions for user "Brian"
+      | ETag | /^"[a-f0-9:\.]{1,32}"$/ |
+    And the content of file "/FOLDER/textfile.txt" for user "Alice" should be:
+    """
+    new description
+    """
+
+
+  Scenario: Uploading a file to a folder received as a upload-only group share
+    Given group "grp1" has been created
+    And user "Brian" has been added to group "grp1"
+    And user "Alice" has created folder "FOLDER"
+    And user "Alice" has created a share with settings
+      | path        | FOLDER |
+      | shareType   | group  |
+      | permissions | create |
+      | shareWith   | grp1   |
+    And user "Brian" has accepted share "/FOLDER" offered by user "Alice"
+    When user "Brian" uploads a file inside space "Shares Jail" with content "new description" to "/FOLDER/textfile.txt" using the WebDAV API
+    Then the HTTP status code should be "201"
+    And the following headers should match these regular expressions for user "Brian"
+      | ETag | /^"[a-f0-9:\.]{1,32}"$/ |
+    And the content of file "/FOLDER/textfile.txt" for user "Alice" should be:
+    """
+    new description
+    """  
+
+
+  Scenario:  Uploading a file to a folder received as a read/write user share
+    Given user "Alice" has created folder "FOLDER"
+    And user "Alice" has created a share with settings
+      | path        | FOLDER |
+      | shareType   | user   |
+      | permissions | change |
+      | shareWith   | Brian  |
+    And user "Brian" has accepted share "/FOLDER" offered by user "Alice"
+    When user "Brian" uploads a file inside space "Shares Jail" with content "new description" to "/FOLDER/textfile.txt" using the WebDAV API
+    Then the HTTP status code should be "201"
+    And the content of file "/FOLDER/textfile.txt" for user "Alice" should be:
+    """
+    new description
+    """  
+
+
+  Scenario: Uploading a file to a folder received as a read/write group share
+    Given group "grp1" has been created
+    And user "Brian" has been added to group "grp1"
+    And user "Alice" has created folder "FOLDER"
+    And user "Alice" has created a share with settings
+      | path        | FOLDER |
+      | shareType   | group  |
+      | permissions | change |
+      | shareWith   | grp1   |
+    And user "Brian" has accepted share "/FOLDER" offered by user "Alice"
+    When user "Brian" uploads a file inside space "Shares Jail" with content "new description" to "/FOLDER/textfile.txt" using the WebDAV API
+    Then the HTTP status code should be "201"
+    And the content of file "/FOLDER/textfile.txt" for user "Alice" should be:
+    """
+    new description
+    """  
+    
+
+  Scenario: Check quota of owners parent directory of a shared file
+    Given user "Admin" has changed the quota of the "Brian Murphy" space to "1"
```

Comment:
I don't understand this test. Why are we creating the space with a quota, but never use that space?

Reply:
Sorry my mistake, these are tests transferred from the core so I changed them exactly as they were there.The quota change should be of `Shares Jail` in this case. I couldn't change the quota of shares jail though I don't know if that's even possible. Do you have any idea?

Reply:
I've also created a issue here https://github.com/owncloud/ocis/issues/4472

Reply:
I think this scenario is not relevant in `OCIS` for `spaces`. In core this scenario sets the quota of the user `Brian` 0 and tests if the file can be uploaded in the received share from `Alice`. But here since `personal` space and `share jails` is a different thing. I don't know how we can test this scenario here.